### PR TITLE
Products

### PR DESCRIPTION
**Pull request by bossmonkey98**
Update: Products lis
Apex Store — Mozilla Firefox 18-02-2022 20_29_35
t

### CODE REVIEW

**Comment by Ranjeetk489** [comment]
if i were to visit the website i would prefer to see more items in one row of listing and the navbar to be bit lesser in width and for the filter section the margin could be more to make it appear seperated as your filter types(subcategories) have more margin than that.

File: components/pages/prodcuts/products.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    display: flex;
+    flex-direction: column;
+    row-gap: 1rem;
+    padding:3rem;
+    border-right: 1px solid var(--primary-color);
+}
+
+.clear{
+    margin-left: 6rem;
+    font-weight: 600;
```
Comment (Ranjeetk489) [inline components/pages/prodcuts/products.css:19]: can use var

**Review by Ranjeetk489 — APPROVED** [review]
Code looks clean and structured, good to merge on that aspect

File: components/pages/aunthentication/auth.css [hunk 2]
```diff
@@ -35,12 +37,18 @@
     margin: 1.5rem;
     width: 8rem;
     color: black;
+    cursor: pointer;
 }
 
 .sign-up{
     color:tomato;
```
Comment (rutu14) [inline components/pages/aunthentication/auth.css:44]: For colors,  use variables to assign them.

File: style.css [hunk 3]
```diff
@@ -1,7 +1,7 @@
 :root{
-    --primary-color:#ee2020;
+    --primary-color:red;
     --secondary-color:rgb(245, 245, 245);
-    --bg-color:rgba(54, 51, 51, 0.575)
+    --bg-color:rgba(37, 37, 37, 0.589);
 }
```
Comment (rutu14) [inline style.css:5]: Use only one way to keep it consistent.

**Review by rutu14 — APPROVED** [review]

File: style.css (hunk 3 above)
Comment (bossmonkey98) [reply style.css:5]: In order to increase the contrast of Text I had to change the background colour

**Review by t007rushi — APPROVED** [review]
Host on netlify else looks good

**Pull request merged**